requirements.txt

```diff
@@ -1 +1,3 @@
 boto3
```

Comment:
Is this requirement still needed? It looks like in #5 all of the AWS related fu was removed, but there appear to be lingering elements like the `terraform` directory and this requirement. You've also removed the `boto3` installation performed in `molecule/default/prepare.yml`.

Reply:
I think you are correct once again.  I will test and confirm.

Reply:
See commits 3e5a3f1 and e70dfc1.

Reply:
Nice 👍 Thanks for checking.